pygradle-plugin/src/main/groovy/com/linkedin/gradle/python/extension/PythonDetails.java

```diff
@@ -127,10 +126,10 @@ public String normalizePythonVersion(String pythonVersion) {
 
         /* Ensure that it's okay to use this version (major/minor only) of Python. */
         System.out.println("PV:" + pythonVersion
-                           + " WL:" + Arrays.asList(PythonVersion.whitelistedPythonVersions)
+                           + " WL:" + PythonVersion.whitelistedPythonVersions
                            + " M.N:" + new PythonVersion(pythonVersion).getPythonMajorMinor());
 
```

Comment:
Yep, I mentioned that in the comment above.  Will definitely remove with a following commit.